lib/htmltoword/xslt/links.xslt

```diff
@@ -0,0 +1,35 @@
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+                xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+                xmlns:o="urn:schemas-microsoft-com:office:office"
+                xmlns:v="urn:schemas-microsoft-com:vml"
+                xmlns:WX="http://schemas.microsoft.com/office/word/2003/auxHint"
+                xmlns:aml="http://schemas.microsoft.com/aml/2001/core"
+                xmlns:w10="urn:schemas-microsoft-com:office:word"
+                xmlns:pkg="http://schemas.microsoft.com/office/2006/xmlPackage"
+                xmlns:msxsl="urn:schemas-microsoft-com:xslt"
+                xmlns:ext="http://www.xmllab.net/wordml2html/ext"
+                xmlns:java="http://xml.apache.org/xalan/java"
+                xmlns:str="http://exslt.org/strings"
+                xmlns:func="http://exslt.org/functions"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                version="1.0"
+                exclude-result-prefixes="java msxsl ext w o v WX aml w10"
+                extension-element-prefixes="func">
+
+  <xsl:template match="a[starts-with(@href, 'http://') or starts-with(@href, 'https://')]" name="link">
```

Comment:
Hey! 
Great stuff! Thanks :blush: 

However... this named template seems to not be working as expected. It is creating the ```<w:hyperlink>```node for local links as well. Therefore, documents cannot be opened since it doesn't find the respective relation. 

Quick example 
*HTML*
```
<!DOCTYPE html>
<html>
<head><title>TESTING</title></head>
<body>
<a href="/link1">Link 1</a>
<a href="http://link2">Link 2</a>
<div>
  <a href="/link3">Link 3</a>
  <a href="http://link4">Link 4</a>
</div>
</body>
</html>
```

Generates
*document.xml*
``` 
<w:p>
  <w:hyperlink r:id="rId8">
    <w:r>
      <w:rPr>
        <w:rStyle w:val="Hyperlink"/>
        <w:color w:val="000080"/>
        <w:u w:val="single"/>
      </w:rPr>
      <w:t xml:space="preserve">Link 1</w:t>
    </w:r>
  </w:hyperlink>
</w:p>
<w:p>
  <w:hyperlink r:id="rId9">
    <w:r>
      <w:rPr>
        <w:rStyle w:val="Hyperlink"/>
        <w:color w:val="000080"/>
        <w:u w:val="single"/>
      </w:rPr>
      <w:t xml:space="preserve">Link 2</w:t>
    </w:r>
  </w:hyperlink>
</w:p>
<w:p>
  <w:r>
    <w:t xml:space="preserve">Link 3</w:t>
  </w:r>
  <w:hyperlink r:id="rId11">
    <w:r>
      <w:rPr>
        <w:rStyle w:val="Hyperlink"/>
        <w:color w:val="000080"/>
        <w:u w:val="single"/>
      </w:rPr>
      <w:t xml:space="preserve">Link 4</w:t>
    </w:r>
  </w:hyperlink>
</w:p>
``` 

*_rels/document.xml.rels* 
```
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
  <Relationship Id="rId3" Type="http://schemas.microsoft.com/office/2007/relationships/stylesWithEffects" Target="stylesWithEffects.xml"/>
  <Relationship Id="rId4" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/settings" Target="settings.xml"/>
  <Relationship Id="rId5" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/webSettings" Target="webSettings.xml"/>
  <Relationship Id="rId6" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/fontTable" Target="fontTable.xml"/>
  <Relationship Id="rId7" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/theme" Target="theme/theme1.xml"/>
  <Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/numbering" Target="numbering.xml"/>
  <Relationship Id="rId2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/styles" Target="styles.xml"/>
  <Relationship Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="http://link2" TargetMode="External" Id="rId9"/>
  <Relationship Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="http://link4" TargetMode="External" Id="rId11"/>
</Relationships>

```